src/components/Local.vue

```diff
@@ -27,43 +11,248 @@
         </v-col>
       </v-row>
 
-      <v-row class="center">
-        <v-btn @click="onSubmit" class="ma-2" outlined large fab color="indigo">
-          <v-icon dark>mdi-plus</v-icon>
-        </v-btn>
+      <v-row class="md-2 center">
+        <v-tooltip left>
+          <template v-slot:activator="{ on }">
+            <v-btn
+              @click="onSubmit"
+              x-large
+              icon
+              class="mr-2 myIcon"
+              fab
+              color="blue"
+              v-on="on"
+            >
+              <v-icon>mdi-plus-circle-outline</v-icon>
+            </v-btn>
+          </template>
+          <span>Add Clip</span>
+        </v-tooltip>
```

Comment:
![image](https://user-images.githubusercontent.com/45892659/82743625-3c308e80-9d6e-11ea-95eb-7f2437d38069.png)


This is how it looks like